Code/Framework/AzCore/Tests/Serialization/Json/PathSerializerTests.cpp

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzCore/IO/Path/Path.h>
+#include <AzCore/IO/Path/PathReflect.h>
+#include <AzCore/Serialization/Json/PathSerializer.h>
+#include <Tests/Serialization/Json/BaseJsonSerializerFixture.h>
+#include <Tests/Serialization/Json/JsonSerializerConformityTests.h>
+
+namespace JsonSerializationTests
+{
+    template<typename PathType>
+    class PathTestDescription
+        : public JsonSerializerConformityTestDescriptor<PathType>
+    {
+    public:
+        using JsonSerializerConformityTestDescriptor<PathType>::Reflect;
+        void Reflect(AZStd::unique_ptr<AZ::SerializeContext>& serializeContext) override
+        {
+            AZ::IO::PathReflect(serializeContext.get());
+        }
+        void Reflect(AZStd::unique_ptr<AZ::JsonRegistrationContext>& jsonContext) override
+        {
+            AZ::IO::PathReflect(jsonContext.get());
+        }
+        AZStd::shared_ptr<AZ::BaseJsonSerializer> CreateSerializer() override
+        {
+            return AZStd::make_shared<AZ::JsonPathSerializer>();
+        }
+
+        AZStd::shared_ptr<PathType> CreateDefaultInstance() override
+        {
+            return AZStd::make_shared<PathType>();
+        }
+
+        AZStd::shared_ptr<PathType> CreateFullySetInstance() override
+        {
+            return AZStd::make_shared<PathType>("O3DE/Relative/Path");
+        }
+
+        AZStd::string_view GetJsonForFullySetInstance() override
+        {
+            return R"("O3DE/Relative/Path")";
```

Comment:
Should "O3DE/Relative/Path" be constexpr declared somewhere to avoid hardcoding and repeating it?

Reply:
It can be declared constexpr, but It is only used in those 2 locations though.